fe/fe-core/src/main/java/org/apache/doris/nereids/rules/implementation/LogicalOlapScanToPhysicalOlapScan.java

```diff
@@ -44,17 +44,19 @@
 public class LogicalOlapScanToPhysicalOlapScan extends OneImplementationRuleFactory {
     @Override
     public Rule build() {
-        return logicalOlapScan().then(olapScan ->
-            new PhysicalOlapScan(
+        return logicalOlapScan().thenApply(ctx -> {
+            LogicalOlapScan olapScan = ctx.root;
+            return new PhysicalOlapScan(
+                    ctx.statementContext.generator.getNextId(),
```

Comment:
we need to generate the id when bind relation. and let logical and physical relation has the same id